.github/workflows/run_eslint.yml

```
@@ -0,0 +1,32 @@
+name: Linting client code
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        working-directory: ./client
+        run: npm ci
+
+      - name: Run linters
+        working-directory: ./client
+        run : npm run lint-fix
```

Minor nit and then we are ready to go

```suggestion
        run : npm run lint
```